src/Attributes/Route.php

```diff
@@ -64,6 +65,11 @@ public function getGetParams(): array
         return $this->getParams;
     }
 
+    /**
+     * @param Parameter[][] $getParams
+     * @param ReflectionAttribute[] $pathParameters
+     * @return void
+     */
```

Comment:
Not sure this is necessary. We can simply put the following if you think it's relevant to the readability of the code
```
/**
 * @param Parameter[][] $getParams
 */
```

Reply:
Oh, that was automaticly added when I was testing, the new parameter is removed, but I didn't remove the autogenerated phpdoc.